cli/cli.go

```diff
@@ -284,6 +286,17 @@ func fetchIndex() {
 	}
 }
 
+func installFromVersionFile() bool {
+	version, err := ioutil.ReadFile(".ruby-version")
+	if err != nil {
+		return false
+	}
+
+	fmtc.Println("Installing version %s from .ruby-version", version)
+	installCommand(string(version))
```

Comment:
Yes, looks like it needs to handle a lot more situations :(

https://github.com/rbenv/rbenv/blob/master/test/version-file-read.bats